src/components/JobSummary/Summary/Summary.js

```diff
@@ -18,8 +20,25 @@ const extractStats = (summary) => {
     }
 }
 
-const Summary = ({ summary, importType }) => {
-    // gml import type object return
+const Summary = ({ summary, importType, isDryRun }) => {
+    if (isDryRun) {
```

Comment:
this logic looks like it would apply to all import dry runs, but we want to only apply it to `tracked entities` and `events` .. right?